File: starting_filesets/C#/Bowling Game/BowlingGameTest.cs

using NUnit.Framework;

[TestFixture]
public class BowlingGameTest
{
    [Test]
    public void Example()
    {
        int expected = 6 * 9;
        int actual = new BowlingGame().Answer;
        Assert.AreEqual(expected, actual);
    }
}

File: starting_filesets/PHP/Bowling Game/BowlingGameTest.php
<?php

require_once 'PHPUnit/Framework.php';
require_once 'BowlingGame.php';

class BowlingGameTest extends PHPUnit_Framework_TestCase
{
    public function testAnswer()
    {
        $this->assertEquals(6 * 9, answer());
    }
}

?>
